app/services/cache/ExportsCacheModelRepository.scala

```diff
@@ -88,5 +98,21 @@ case class ExportsCacheModel(
 )
 
 object ExportsCacheModel {
-  implicit val format = Json.format[ExportsCacheModel]
+  implicit val formatInstant: OFormat[LocalDateTime] = new OFormat[LocalDateTime] {
```

Comment:
This will mean we have to clear the DB down

Reply:
As  discussed in other PRs we shouldn't really be using LocalDateTime as it doesnt contain time zone info. Instant would be preferred (which is what Mongo uses)